Jenkinsfile

```diff
@@ -274,11 +274,6 @@ pipeline {
           def processor = new ResultsProcessor()
           processor.processResults(mapResults)
           archiveArtifacts allowEmptyArchive: true, artifacts: 'results.json,results.html', defaultExcludes: false
-          catchError(buildResult: 'SUCCESS') {
-            def datafile = readFile(file: "results.json")
-            def json = getVaultSecret(secret: 'secret/apm-team/ci/apm-server-benchmark-cloud')
-            sendDataToElasticsearch(es: json.data.url, data: datafile, restCall: '/jenkins-builds-test-results/_doc/')
```

Comment:
Should we [modify the library](https://github.com/elastic/apm-pipeline-library/blob/master/vars/sendDataToElasticsearch.groovy) to raise a warning if this is called? (Or just remove it?) I don't see any mention of it being deprecated.

Reply:
The step is not deprecated but the call in this particular pipeline. `sendDataToElasticsearch` is still a valid step and it's in fact used by the `notifyBuildResult` step to send data to ES.

The `jenkins-builds-test-results` index is not something we are using anymore, and used to collect the status of the downstream jobs for the current pipeline and store their results, but not the process we wanna keep using from now on.

Reply:
Ahh, OK. I misunderstood what you meant originally. Thanks for the clarification!